SConstruct

```diff
@@ -15,7 +15,7 @@ try:
 except NameError:
     pass
 
-sys.path.insert(0, os.path.join(Dir('#').abspath, 'utils'))
+sys.path.insert(0, os.path.join(Dir('#').abspath, 'utils/sl'))
```

Comment:
Is there a reason to move this to sl?

Reply:
from the commit message, "and put daos_build in the same directory as env_helper". We have two python helper scripts that are imported, and two sys.path.insert lines in different parts of the code.  I'm just trying to put everything in the same place.